teleport-node/TAG

```diff
@@ -1 +1 @@
-14.3.0.3
+14.3.0.4
```

Comment:
Why is teleport-node tag updated?

Reply:
@yamatcha 
Sorry for confusion, it is not relevant to meows.
Just I wanted to install column command to Teleport nodes this time.

```
[2024-02-15 11:26] 8a10d734 by Daichi Sakaue: Update teleport-node to include column command
[2024-02-15 11:23] d2cb16a5 by Daichi Sakaue: Make meows-runner images use a base-image from ghcr.io
```